players/group5/player_map.py

```diff
@@ -438,19 +438,27 @@ def get_valid_moves(self, turn_num: int) -> List[int]:
 
     def get_wall_freq_candidates(self, door_id: DoorIdentifier) -> List[Set[int]]:
         door_freq_candidates = self._get_freq_candidates_usecase(door_id.absolute_coord, door_id.door_type)
+
+        # print("Inner Door freq candidates: ", door_freq_candidates)
+        # print("Door ID: ", door_id)
         
         door_type_to_touching_door_offsets = {
-            constants.LEFT: ([0, -1], constants.RIGHT),
-            constants.UP: ([-1, 0], constants.DOWN),
-            constants.RIGHT: ([0, 1], constants.LEFT),
-            constants.DOWN: ([1, 0], constants.UP),
+            constants.LEFT: ([-1, 0], constants.RIGHT),
```

Comment:
@Yoni-Livstone  wouldn't this point to the door in the cell above facing right instead of the door on the left facing right?

Reply:
No I don't believe so. I beleive this is a logic mistake. You move left and want to look at the right door.